ContactDetailsApi/V2/Gateways/ContactDetailsDynamoDbGateway.cs

```diff
@@ -101,5 +114,344 @@ public async Task<ContactDetails> CreateContact(ContactDetailsEntity contactDeta
 
             return contactDetails.ToDomain();
         }
+
+
+
+        public async Task<List<ContactDetailsEntity>> FetchAllContactDetails()
+        {
+            var rawResults = new List<Document>();
+
+            var table = Table.LoadTable(_dynamoDB, "ContactDetails");
+
+            var scan = table.Scan(new ScanOperationConfig
+            {
+
+            });
+
+            do
+            {
+                try
+                {
+                    var newResults = await scan.GetNextSetAsync();
+                    rawResults.AddRange(newResults);
+                }
+                catch
+                {
+                    //throw;
+                }
+            }
+            while (!scan.IsDone);
+
+            var results = new List<ContactDetailsEntity>();
+
+            foreach (var result in rawResults)
+            {
+                try
+                {
+                    var id = result["id"];
+                    var targetId = result["targetId"];
+                    var rawContactInformation = result["contactInformation"].AsDocument();
+                    var contactValue = rawContactInformation.ContainsKey("value") ? rawContactInformation["value"] : null;
+                    var contactInformation = new ContactInformation
+                    {
+                        Value = contactValue ?? ""
+                    };
+
+                    var isActive = result["isActive"];
+
+
+                    var entity = new ContactDetailsEntity
+                    {
+                        Id = (Guid) id,
+                        TargetId = (Guid) targetId,
+                        ContactInformation = contactInformation,
+                        IsActive = (bool) isActive
+                    };
+
+                    results.Add(entity);
+                }
+                catch
+                {
+                    //throw;
```

Comment:
If the intention here is to continue execution if there is an exception thrown in the try-block, you might need to call `continue;` in the catch-block

Reply:
I have removed this as no longer required